public/views/partials/footer.html

```diff
@@ -0,0 +1,17 @@
+<footer id="footer">
```

Comment:
I dunno. I thought that since there would only be one footer, it wouldn't matter if I used a unique id name.